locale/da_DK/emails.po

```diff
@@ -32,19 +32,8 @@ msgstr ""
 "<br />\n"
 "Mvh. {$siteContactName}"
 
-msgid "emails.passwordReset.subject"
-msgstr "Nulstil adgangskode"
-
-msgid "emails.passwordReset.body"
-msgstr ""
-"Din adgangskode er blevet nulstillet og kan benyttes på webstedet "
-"{$siteTitle}. Opbevar brugernavnet og adgangskoden, da de er nødvendige for "
-"alt arbejde i forbindelse med tidsskriftet.<br />\n"
-"<br />\n"
-"Dit brugernavn: {$recipientUsername}<br/>\n"
-"Adgangskode: {$password}<br />\n"
-"<br />\n"
-"Mvh. {$siteContactName}"
+msgid "emails.passwordResetConfirm.description"
```

Comment:
@touhidurabir, I think this is an accidental re-add of an obsolete locale key.

Reply:
Removed.